gtk4/sample/examples/column_view_demo.rb

```diff
@@ -0,0 +1,76 @@
+# Standalone demo of ColumnView in gtk4.
+
+require "gtk4"
+
+class DataObject < GLib::Object
+    attr_accessor :fruit, :color
+    def initialize(fruit, color)
+      super()
+      @fruit = fruit
+      @color = color
+    end
+end
+
+def setup(factory, cell)
+    # Setup the widget to show in the Gtk.Listview
+    label = Gtk::Label.new()
```

Comment:
Could you remove `()` from no argument method calls?

Reply:
I actually like those because it tells me that its a method.

Reply:
Dude, this is your project.  Do whatever you want to this file.